### ISSUE TITLE
Request: Fiber

### ISSUE THREAD

**Issue by antegral** [simple-icons/simple-icons#8170]
### Brand Name

fiber

### Website

https://gofiber.io/

### Popularity Metric

[31.6k stars on GitHub.](https://github.com/gofiber/fiber)

### Official Resources for Icon and Color

SVG: https://github.com/gofiber/docs/blob/master/static/img/logo.svg
Color: `#00ACD7`, from the badge

### Contributing

No

**Comment by uncenter** [comment]
In scope, has enough GitHub stars and is still active.

**Comment by merge-conflict-XoX** [comment]
Hi, I'm a first time contributor. Is this still open? Can I work on it?

**Comment by antegral** [comment]
Yes, I had paused this work. Since you've already completed it, I'll close this issue.

**Comment by antegral** [comment]
Instead of closing this issue, I think it's better to change the Contributing status to No, so I reopened the issue and edited the post.

Therefore, yes, you may proceed.
